src/Core/Resources/SharedResources.en.resx

```diff
@@ -587,6 +587,15 @@
   <data name="PersonalOwnershipExemption" xml:space="preserve">
     <value>Organization Owners and Administrators are exempt from this policy's enforcement.</value>
   </data>
+  <data name="DisableSend" xml:space="preserve">
+    <value>Disable Send</value>
+  </data>
+  <data name="DisableSendDescription" xml:space="preserve">
+    <value>Do not allow users to create or edit Bitwarden Sends. Deleting existing Sends is still allowed</value>
+  </data>
+  <data name="DisableSendExemption" xml:space="preserve">
+    <value>Organization Owners and Administrators are exempt from this policy's enforcement.</value>
+  </data>
```

Comment:
I see there is a standard for "Owners and Administrators" verbiage here, but we should probably change this to account for the ManagePolicies permission to.

```suggestion
  <data name="DisableSendExemption" xml:space="preserve">
    <value>Organization members that manage policies are exempt from this policy's enforcement.</value>
  </data>
```

Reply:
I like this change, but I'm going to hold off on making it on all policies since we don't use this helper everywhere.

I'm going to make an issue to permeate this change, though.